__tests__/integration.test.js

```diff
@@ -35,14 +33,60 @@ describe("GET: /api/invalid_endpoint", () => {
   });
 });
 describe("GET: /api", () => {
-  test("Should return a JSON object for all articles", () => {
+  test("Should return a JSON object for all endpoints", () => {
+    return request(app)
+      .get("/api/")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.endpoint).toEqual(expectedContents);
+      });
+  });
+});
+describe("GET: /api/articles/:article_id", () => {
+  test("GET: responds with a 200 status code and a nested object", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({body}) => {
+        expect(body).toBeInstanceOf(Object)
+        expect(body.articles).toBeInstanceOf(Object)
```

Comment:
Arrays are instances of Objects, so this is passing, but the response is currently an array on the key of `articles`
But ideally we want the response to be a single object on the key of `article`